ext/src/dll/opentelemetry_cpp.src

```diff
@@ -0,0 +1,32 @@
+// clang-format off
```

Comment:
- If this is checked-in, we can add the copyright
- How is this file different from the def file, as there are two different extensions used in the PR - def and src ?
- Do we need def file explicitly when we are already specifying `__declspec(dllexport)` symbols in the code ?

Reply:
* Added copyright.
* .src is preprocessed into .def.
* Adding to .src/.def file is needed if there is no direct reference to the class/function. `__declspec(dllexport)` doesn't guarantee the class/function will be linked to the DLL if it is not invoked.